src/20ec0_len_5040.c

```diff
@@ -2,28 +2,174 @@
 #include "map.h"
 
 extern s32 D_800A0F40;
+extern PopupMessage D_800A0BC0[32];
+
+void update_merlee_message(PopupMessage* popup);
+void draw_merlee_message(PopupMessage* popup);
+void func_80045BC8(void);
+void func_80045FB4(void);
+
+extern s16 D_80078160[];
+extern s32 D_80078168[];
```

Comment:
Is there a reason why these are down here instead of being grouped with the other externs at the top of the file?